auth.config.ts

```diff
@@ -1,9 +1,9 @@
+import { PrivateKeyWallet } from "@thirdweb-dev/auth/evm";
 import { ThirdwebAuth } from "@thirdweb-dev/auth/next";
 
 export const { ThirdwebAuthHandler, getUser } = ThirdwebAuth({
-  // Using environment variables to secure your private key is a security vulnerability.
-  // Learn how to store your private key securely:
-  // https://portal.thirdweb.com/sdk/set-up-the-sdk/securing-your-private-key
-  privateKey: process.env.ADMIN_PRIVATE_KEY as string,
-  domain: "example.com",
+  domain: process.env.NEXT_PUBLIC_AUTH_DOMAIN || "",
+  wallet: new PrivateKeyWallet(process.env.ADMIN_PRIVATE_KEY || ""),
```

Comment:
Lets change the names of these environment variables to the ones we've been using elsewhere `NEXT_PUBLIC_THIRDWEB_AUTH_DOMAIN` and `THIRDWEB_AUTH_PRIVATE_KEY`